packages/core/echo/echo/src/api/type.ts

```diff
@@ -0,0 +1,128 @@
+//
+// Copyright 2025 DXOS.org
+//
+
+import { type Schema } from 'effect';
+
+import { type AnyLiveObject as AnyLiveObject$ } from '@dxos/echo-db';
+import {
+  type BaseEchoObject,
+  type BaseObject,
+  type BaseSchema,
+  type EchoSchema,
+  EchoObject,
+  EntityKind,
+  Expando as Expando$,
+  type ImmutableSchema,
+  type JsonSchemaType,
+  ObjectId as ObjectId$,
+  Ref as Ref$,
+  SpaceIdSchema,
+  type StoredSchema,
+  type TypeMeta,
+  getTypeAnnotation,
+  getSchema as getSchema$,
+  getSchemaDXN,
+  getSchemaTypename,
+  getSchemaVersion,
+  isInstanceOf,
+  toJsonSchema as toJsonSchema$,
+} from '@dxos/echo-schema';
+import { invariant } from '@dxos/invariant';
+import { type SpaceId as SpaceId$ } from '@dxos/keys';
+import { live as live$ } from '@dxos/live-object';
+
+// TODO(burdon): Type vs. Relation vs. Object.
+
+/**
+ * Type System API.
+ *
+ * @category api namespace
+ * @since 0.9.0
+ */
+export namespace Type {
```

Comment:
currently this will force us to use `Type.create` for relations. Let's figure out where to put the verbs which are usable with both objects and relations